pkg/mdformatter/linktransformer/link.go

```diff
@@ -241,7 +257,8 @@ func NewValidator(ctx context.Context, logger log.Logger, linksValidateConfig []
 	}
 	if err := v.c.Limit(&colly.LimitRule{
 		DomainGlob:  "*",
-		Parallelism: 100,
+		Parallelism: 10,
```

Comment:
> The parallelism configuration of colly has been brought down from 100
to 10. This setting means that at most 10 requests will be sent in
parallel for the matching domains, instead of 100. This way we are
more friendly to the servers and hopefully they will be happier with us too,
returning more responses with 200 status code faster.

Hope based development 🙈 

I think this makes sense generally, but I wonder if it makes sense to perhaps expose those options so we can tune it to every environment. Right now we are bit guessing with programmatic change that takes time to modify and release.

Reply:
Better, I am moving them to the config file, like the timeout.

Reply:
Added some parameters in the config file via https://github.com/bwplotka/mdox/pull/86/commits/23f1deca397085457bb4616ab93120a1175d6125.